scene/3d/mesh_instance_3d.cpp

```diff
@@ -87,17 +87,9 @@ bool MeshInstance3D::_get(const StringName &p_name, Variant &r_ret) const {
 }
 
 void MeshInstance3D::_get_property_list(List<PropertyInfo> *p_list) const {
-	List<String> ls;
-	for (const KeyValue<StringName, int> &E : blend_shape_properties) {
-		ls.push_back(E.key);
-	}
-
-	ls.sort();
-
-	for (const String &E : ls) {
-		p_list->push_back(PropertyInfo(Variant::FLOAT, E, PROPERTY_HINT_RANGE, "-1,1,0.00001"));
+	for (uint32_t i = 0; i < blend_shape_tracks.size(); i++) {
+		p_list->push_back(PropertyInfo(Variant::FLOAT, vformat("blend_shapes/%s", String(mesh->get_blend_shape_name(i))), PROPERTY_HINT_RANGE, "-1,1,0.00001"));
```

Comment:
```suggestion
	for (const KeyValue<StringName, int> &E : blend_shape_properties) {
		p_list->push_back(PropertyInfo(Variant::FLOAT, E.key, PROPERTY_HINT_RANGE, "-1,1,0.00001"));
	}
```
Isn't enough this? Since `blend_shape_properties` order is already depends on `blend_shape_tracks` by `MeshInstance3D::_mesh_changed()`.

Reply:
As far as I know `HashMap<StringName, int> blend_shape_properties` is unsorted, so no.